apps/core/client/queries/get-blog-posts.ts

```diff
@@ -31,15 +31,16 @@ const GET_BLOG_POSTS_QUERY = /* GraphQL */ `
           name
           posts(first: $first, after: $after, last: $last, before: $before, filters: $filters) {
             pageInfo {
-              ...PageDetails
+              hasNextPage
+              hasPreviousPage
+              startCursor
+              endCursor
```

Comment:
We don't want to use `PageDetails` fragment anymore?

Reply:
Trying to use the least amount of shared fragments for now. Until we get to colocation.